cmd/controller/main.go

```diff
@@ -266,6 +278,7 @@ func main() {
 	kubeInformerFactory.Start(stopCh)
 	projectriffInformerFactory.Start(stopCh)
 	knbuildInformerFactory.Start(stopCh)
+	kedaInformerFactory.Start(stopCh)
```

Comment:
we don't want to require that Keda be installed for all runtimes, much like we don't want Knative Serving to be required for all runtimes.

Reply:
Implemented via f05f64b